Tests/src/test/scala/equellatests/tests/LoginNoticeMenuProperties.scala

```diff
@@ -0,0 +1,90 @@
+package equellatests.tests
+
+import equellatests.ShotProperties
+import equellatests.domain.RandomWord
+import equellatests.instgen.fiveo.autoTestLogon
+import equellatests.pages.{LoginNoticePage, LoginPage}
+import org.openqa.selenium.By
+import org.scalacheck.Prop
+import org.scalacheck.Prop.forAll
+
+object LoginNoticeMenuProperties extends ShotProperties("Login Notice Menu Properties") {
+
+  property("pre login notice creation") = forAll { w1: RandomWord =>
+    withLogon(autoTestLogon) { context =>
+      val page = LoginNoticePage(context).load()
+      val notice = s"${w1.word}"
+      page.populatePreNoticeField(notice)
+      page.clickPreApply()
+      page.load()
+      Prop(page.getPreNoticeFieldContents == notice).label("Notice: " + notice + ", NoticeField: " + page.getPreNoticeFieldContents)
+    }
+  }
+
+  property("post login notice creation") = forAll { w1: RandomWord =>
+    withLogon(autoTestLogon) { context =>
+      val page = LoginNoticePage(context).load()
+      page.clickPostTab()
+      val notice = s"${w1.word}"
+      page.populatePostNoticeField(notice)
+      page.clickPostApply()
```

Comment:
Generally you will want to wait for something to happen on the page here, as it's relying on the load() to occur after the click has been processed fully.